build.sbt

```diff
@@ -421,8 +421,9 @@ lazy val notificationworkerlambda = lambda("notificationworkerlambda", "notifica
       "com.google.firebase" % "firebase-admin" % "9.0.0",
       "com.google.protobuf" % "protobuf-java" % "3.19.2",
       "com.amazonaws" % "aws-lambda-java-events" % "2.2.8",
-      "com.amazonaws" % "aws-java-sdk-sqs" % awsSdkVersion,
+      "com.amazonaws" % "aws-java-sdk-sqs" % "1.12.338",
```

Comment:
Out of curiosity (most because I can't remember), did we need to bump just this library specifically?

Reply:
I have now bumped all the libraries that rely on the aws sdk version